schema/mysql/schema.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE available_channel_type (
 
 CREATE TABLE channel (
     id bigint NOT NULL AUTO_INCREMENT,
+    external_uuid char(36) NOT NULL, -- used for external references, lower case
```

Comment:
The channel table isn't mentioned in the PR title (or description). If this is intended, please update these as well.